yogstation/code/game/machinery/doors/airlock.dm

```diff
@@ -0,0 +1,28 @@
+/obj/machinery/door/airlock/AltClick(mob/living/user)
+	if(isobserver(user))
+		to_chat(user, "<span class='danger'>Psst, you can't interact with the world, ghosts!</span>")
+		return
+
+	if(!user.canUseTopic(src))
+		to_chat(user, "<span class='info'>You can't do this right now!</span>")
+		return
+
+	if(stat & (NOPOWER|BROKEN) || (obj_flags & EMAGGED))
+		to_chat(user, "<span class='info'>The door isn't working!</span>")
+		return
+
+	if(request_cooldown > world.time)
+		to_chat(user, "<span class='info'>The airlock's spam filter is blocking your request. Please wait at least 10 seconds between requests.</span>")
```

Comment:
I would remove all of the feedback here except for the one that says request sent when it is successful, I don't think it's needed to display a message for every single thing